editoast/core_client/src/stdcm.rs

```diff
@@ -31,6 +33,8 @@ pub struct Request {
     pub path_items: Vec<PathItem>, // FIXME: the schema is referenced in the openapi, not the struct below
     /// The loading gauge of the rolling stock
     pub rolling_stock_loading_gauge: LoadingGaugeType,
+    /// Set of authorized track section ids for the current loading gauge
```

Comment:
As far as I understand, we do develop this feature for loading gauge, but the feature is bound to be more general than that (see https://github.com/OpenRailAssociation/osrd/pull/13709#discussion_r2477158283). Therefore, I would simplify the comment to the following.


```suggestion
    /// Set of authorized track section ids for the current request
```